### PR TITLE
remove "Customize User Registration Form" j2me feature

### PR DESCRIPTION
**Pull request by dannyroberts**
There's a bunch more code we can delete. I'm double-checking with Kevin first (https://trello.com/c/mrl9gPPC/3-user-registration-j2me)

### CODE REVIEW

**Comment by orangejenny** [comment]
YES.

Assume you're talking to Kevin about all the `is_user_registration` conditionals, which would be fabulous to remove.

**Comment by czue** [comment]
will be glorious to have this gone. :city_sunset:

**Comment by dannyroberts** [comment]
Ok I'm feeling pretty good about this. Checked on staging that an older app still builds, clicking around loads, etc.

**Comment by czue** [comment]
the reindex is the `AppPillow`, yeah? I think that that's pretty quick. any reason not to merge this in now?

**Comment by czue** [comment]
@dannyroberts on a :skull: ing :rage1:

**Comment by dannyroberts** [comment]
Yeah it's just app pillow so it's good to merge!

**Comment by czue** [comment]
added a note to the trello, but we should also remove the xml processing code too

File: corehq/apps/app_manager/suite_xml/sections/resources.py [hunk 1]
```diff
@@ -9,34 +9,21 @@ class FormResourceContributor(SectionContributor):
     section_name = 'xform_resources'
```
Comment (dannyroberts) [inline corehq/apps/app_manager/suite_xml/sections/resources.py]: @czue Isn't this file the xml processing part?

Comment (czue) [inline corehq/apps/app_manager/suite_xml/sections/resources.py]: I meant the code that processes a user registration submission and makes a user out of it. I think the entry point is [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/signals.py#L44)

Comment (dannyroberts) [inline corehq/apps/app_manager/suite_xml/sections/resources.py]: Ah, gotcha. Agreed!

**Pull request merged**